api/organisations/subscription_info_cache.py

```diff
@@ -30,14 +30,18 @@ def update_caches():
         for org in organisations
     }
 
-    _update_caches_with_influx_data(organisation_info_cache_dict)
-    _update_caches_with_chargebee_data(organisations, organisation_info_cache_dict)
+    if SubscriptionCacheEntity.INFLUX in update_cache_entities:
+        _update_caches_with_influx_data(organisation_info_cache_dict)
+
+    if SubscriptionCacheEntity.CHARGEBEE in update_cache_entities:
+        _update_caches_with_chargebee_data(organisations, organisation_info_cache_dict)
 
     to_update = []
     to_create = []
 
     for subscription_info_cache in organisation_info_cache_dict.values():
-        subscription_info_cache.updated_at = timezone.now()
+        subscription_info_cache.chargebee_updated_at = timezone.now()
+        subscription_info_cache.influx_updated_at = timezone.now()
```

Comment:
I don't think these should both be updated here, right? We should only update them if the relevant `SubscriptionCacheEntity` was included.

Reply:
Corrected.